src/analysis.ts

```diff
@@ -100,7 +100,6 @@ export async function analyzeFolders(options: FileAnalysisOptions): Promise<File
     ...options.connection,
     ...options.fileOptions,
     languages: options.languages,
-    ...(options.analysisContext ? { analysisContext: options.analysisContext } : {}),
```

Comment:
q: I assume the analysisContext is never used in this flow?

Reply:
> q: I assume the analysisContext is never used in this flow?

yes